wavefront/server/packages/flo_cloud/flo_cloud/postgres/postgres.py

```
@@ -0,0 +1,261 @@
+            try:
+                cursor.execute(query, params)
+                return [dict(row) for row in cursor.fetchall()]
+            except psycopg2.Error as e:
+                logger.error(f'Query execution error: {e}')
+                raise
+
+    def execute_query_to_dict(
+        self,
+        projection: str = '*',
+        table_prefix: str = '',
+        table_names: List[str] = [],
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)